packages/grid/x-data-grid/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../../tsconfig.json",
   "compilerOptions": {
+    "jsx": "react",
```

Comment:
Why this change?

Reply:
A mistake from when I was struggling with the migration